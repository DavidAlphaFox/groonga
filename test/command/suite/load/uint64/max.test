table_create Numbers TABLE_NO_KEY
column_create Numbers value COLUMN_SCALAR UInt64

load --table Numbers
[
{"value": 18446744073709551615}
]

select Numbers
