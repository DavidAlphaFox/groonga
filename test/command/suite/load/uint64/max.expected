table_create Numbers TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Numbers value COLUMN_SCALAR UInt64
[[0,0.0,0.0],true]
load --table Numbers
[
{"value": 18446744073709551615}
]
[[0,0.0,0.0],1]
select Numbers
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "value",
          "UInt64"
        ]
      ],
      [
        1,
        18446744073709551615
      ]
    ]
  ]
]
